pandas/tests/plotting/test_groupby.py

```diff
@@ -10,10 +10,54 @@
 import numpy as np
 
 from pandas.tests.plotting.common import TestPlotBase
+import pytest
 
 
 @td.skip_if_no_mpl
 class TestDataFrameGroupByPlots(TestPlotBase):
+    @pytest.mark.parametrize(
+        'bins, equal_bins',
+        zip([5, None, np.linspace(-3, 3, 10)], [True, False])
+    )
```

Comment:
Trying all possible forms of `bins` with and without `equal_bins`

Reply:
Would still prefer if you could be explicit about what is expected here, i.e. add `expected` as a third parameter here and assert that we get the correct number of bins across the two groups